hkmc2/shared/src/test/mlscript/HkScratch.mls

```diff
@@ -4,4 +4,9 @@
 // :elt
 
 
-
+:parseOnly
+(let x = 1) + 1
+x
```

Comment:
Tis should be moved to another test file. Probably `hkmc2/shared/src/test/mlscript/parser/Let.mls`.

Reply:
(We try to keep the scratch file empty.)

Reply:
my bad, forgot to remove it before my commit >_>